versions/development/SPEC.md

```diff
@@ -760,7 +760,9 @@ Engines should at the very least support the following protocols for import URIs
 
 * `http://` and `https://`
 * `file://`
-* no protocol (which should be interpreted as `file://`
+* No protocol (see below)
```

Comment:
should probably bump the minor version

Reply:
Of the WDL spec? this is currently against `versions/development`

Reply:
I had the same question @yfarjoun - regardless, yes this would be for WDL 1.1.